academic-services/knowledge-exploration-service/tutorial-conference-filterable-paperlist.md

```diff
@@ -0,0 +1,382 @@
+    ```
+
+For more detailed deployment instructions, See [Create API Instances](get-started-create-api-instances.md#create-makes-hosting-resources)
+
+> [!NOTE]
+> Since the index we're hosting is relatively small, you can reduce Azure consumption for the tutorial MAKES host instance by using the "--HostMachineSku" parameter and set the SKU to "Standard_D2_V2".
+
+## Create Client Application with MAKES REST APIs
+
+We now have a backend API to serve our conference paper data. The last step is to create the client application to showcase the filterable paper list. The client application will retrieve data and generate filters via Evaluate and CalcHistogram APIs.
+
+### Paper list KES Query Expression
+
+We start building our client by crafting a KES query expression to represent the paper list shown on the UI. Since the initial list of papers we want to see is "all papers", the corresponding KES query expression would be "**All()**".
+
+This corresponds to the following code in **<tutorial_resource_root>/ConferenceWebsite/index.js**
+
+```javascript
+/*
+ * Client app entry point/ main.
+ */
+var app = new FilterablePaperList();
+app.setOriginalPaperListExpression("All()");
+mount(document.body, app);
+```
+
+We will use this expression to fetch paper data in the next step. When filters are applied, we will modify this expression to get the corresponding data.
+
+For more information on KES Query Expressions, see [Structured query expressions](concepts-query-expressions.md)
+
+### Retrieve top papers
+
+We can call Evaluate API with the paper list expression ( Initially set to "**All()**" ) to retrieve paper entities for display.
+
+To get papers using Evaluate API, see **MakesInteractor.GetPapers(paperExpression)** method in **<tutorial_resource_root>/ConferenceWebsite/makesInteractor.js**:
+
+```javascript
+/*
+* Gets a list of papers using a KES expression
+*/
+async GetPapers(paperExpression)
+{
+    let requestBody = {
+            expr: paperExpression,
+            attributes: this.paperListItemAttributes,
+            count: this.paperListItemCount
+        }
+
+    let response = await Promise.resolve($.post(this.GetEvaluateApiEndpoint(), requestBody));
+    return response?.entities;
+}
+```
+
+For more information on Evaluate API, see [Evaluate REST API](reference-post-evaluate.md)
+
+ After retrieving the paper entities from Evaluate API, all is left to do is to translate the entity data to UI elements. The corresponding data transformation logic for paper UI elements can be found in:
+
+- **<tutorial_resource_root>/ConferenceWebsite/paperListItem.js**
+- **<tutorial_resource_root>/ConferenceWebsite/paperFieldsOfStudyListItem.js**
+
+### Generate filters  
+
+We can also call Histogram API with the paper list expression to get filter attribute histograms and transform them into filters.
```

Comment:
>Histogram [](start = 17, length = 9)

Please do a find and replace for Histogram to CalcHistogram if that is the signature that we expose.